authorization/src/main/java/life/qbic/authorization/application/AppContextProvider.java

```diff
@@ -0,0 +1,21 @@
+package life.qbic.authorization.application;
```

Comment:
Since then navigation is based on URL and the user authorization is not checked in the contextprovider i'd recommend moving the interface from the `authorization `package to the more general `life.qbic.datamanager` package

Reply:
Dependency Inversion Principle. My only concern here was to introduce lose coupling so the auth module is not depending on the app UI module.